test/FluentMigrator.Tests/Integration/MigrationRunnerTests.cs

```diff
@@ -629,6 +623,10 @@ public void CanMigrateASpecificVersionDown()
         {
             try
             {
+                // Can't currently test SQLite as we are using in-memory mode
```

Comment:
@mattbrailsford  I believe this comment is no longer accurate?

Reply:
Removed comment